src/java/org/apache/cassandra/service/accord/AccordCallback.java

```diff
@@ -32,18 +34,22 @@
 class AccordCallback<T extends Reply> implements RequestCallback<T>
 {
     private static final Logger logger = LoggerFactory.getLogger(AccordCallback.class);
+    private final CommandStore commandStore;
     private final Callback<T> callback;
 
-    public AccordCallback(Callback<T> callback)
+    public AccordCallback(CommandStore commandStore, Callback<T> callback)
     {
+        this.commandStore = commandStore;
         this.callback = callback;
     }
 
     @Override
     public void onResponse(Message<T> msg)
     {
         logger.debug("Received response {} from {}", msg.payload, msg.from());
-        callback.onSuccess(EndpointMapping.endpointToId(msg.from()), msg.payload);
+        commandStore.execute(PreLoadContext.empty(), ignore ->
+                                                     callback.onSuccess(EndpointMapping.endpointToId(msg.from()), msg.payload))
+        .begin(commandStore.agent());
```

Comment:
should this align with .execute()? If not, let's bring down the closing brace, so it's clear we're continuing the prior line.